CHANGELOG.md

```diff
@@ -1,4 +1,28 @@
-## 0.17.0 (Unreleased)
+## 0.18.0 (Unreleased)
+
+## 0.17.0 (June 19, 2020)
+
+ENHANCEMENTS:
+
+* resource/fastly_service_v1/logging_googlepubsub: Add Google Cloud Pub/Sub logging support ([#258](https://github.com/terraform-providers/terraform-provider-fastly/pull/258))
+* resource/fastly_service_v1/logging_kafka: Add Kafka logging support ([#254](https://github.com/terraform-providers/terraform-provider-fastly/pull/254))
+* resource/fastly_service_v1/logging_scalyr: Add Scalyr logging support ([#252](https://github.com/terraform-providers/terraform-provider-fastly/pull/252))
+* resource/fastly_service_v1/s3logging: Add support for public key field ([#249](https://github.com/terraform-providers/terraform-provider-fastly/pull/249))
+* resource/fastly_service_v1/logging_newrelic: Add New Relic logging support ([#243](https://github.com/terraform-providers/terraform-provider-fastly/pull/243))
+* resource/fastly_service_v1/logging_datadog: Add Datadog logging support ([#242](https://github.com/terraform-providers/terraform-provider-fastly/pull/242))
+* resource/fastly_service_v1/logging_loggly: Add Loggly logging support ([#241](https://github.com/terraform-providers/terraform-provider-fastly/pull/241))
+* resource/fastly_service_v1/logging_sftp: Add SFTP logging support ([#236](https://github.com/terraform-providers/terraform-provider-fastly/pull/236))
+* resource/fastly_service_v1/logging_ftp: Add FTP logging support ([#235](https://github.com/terraform-providers/terraform-provider-fastly/pull/235))
+* resource/fastly_service_v1/logging_elasticsearch: Add Elasticsearch logging support ([#234](https://github.com/terraform-providers/terraform-provider-fastly/pull/234))
+
+NOTES:
+
+* resource/fastly_service_v1/httpslogging: Use `trimSpaceStateFunc` to trim leading and trailing whitespace from the `tls_ca_cert`, `tls_client_cert` and `tls_client_key` fields ([#264](https://github.com/terraform-providers/terraform-provider-fastly/pull/264))
+* resource/fastly_service_v1/splunk: Use `trimSpaceStateFunc` to trim leading and trailing whitespace from the `tls_ca_cert` field ([#264](https://github.com/terraform-providers/terraform-provider-fastly/pull/264))
```

Comment:
@phamann These could introduce a new diff of state for customers. Would remove a new line.

I don't think this is technically a breaking change, but does introduce a state change just from updating your provider version.

Just something to think about.